### PR TITLE
RND-18678 - Improve Ping Logging & Update Documentation

### PR DESCRIPTION
**Pull request by EliAsor**

### CODE REVIEW

File: examples/responses/schema.md [hunk 1]
```diff
@@ -39,15 +45,22 @@ Responses returned by Verbit's Streaming Speech Recognition services have the fo
 - `"response"` - The root element in the response JSON
   - `"id"` - A unique identifier of the response (UUID)
   - `"type"` - The response type. Can be either "transcript" or "captions" (See explanation in [README.md](https://github.com/verbit-ai/verbit-streaming-python-sdk/blob/main/README.md#responses)).
+  - `"start"` - The start time of the utterance. Measured in seconds from the beginning of the media stream.
+  - `"end"` - The (current) end time of the utterance. Measured in seconds from the beginning of the media stream.
+  - `"start_pts"` - The pts value corresponding to the `"start"` of the response, as received from the input media stream. Measured in seconds.
+    - Note: if the input media stream doesn't provide pts values, this field will always contain `0`.
```
Comment (elad-shtilerman) [inline examples/responses/schema.md:51]:
```suggestion
    - Note: if the input media stream doesn't provide pts values, this field will default to `0`.
```

**Review by elad-shtilerman — APPROVED** [review]

File: verbit/streaming_client.py [hunk 2]
```diff
@@ -528,7 +543,7 @@ def _handle_socket_close(self, data):
             if code not in (STATUS_NORMAL, STATUS_GOING_AWAY):
                 self._logger.warning('Unexpected close code: ' + msg)
             else:
-                self._logger.debug(msg)
+                self._logger.info(msg)
```
Comment (elad-shtilerman) [inline verbit/streaming_client.py:547]: @EliAsor In any case, don't we need to close the web socket here too, as in `_close_ws`?

**Comment by elad-shtilerman** [comment]
Also, let's rename the PR so later the WS changes are reflected in the commit message, etc.

File: verbit/streaming_client.py (hunk 2 above)
Comment (EliAsor) [reply verbit/streaming_client.py:547]: As we discussed f2f, the underlying websocket client will have already sent the "close" reply message to the server by this point, so we don't need to do anything else to close the connection here.

**Review by elad-shtilerman — APPROVED** [review]

**Review by elad-shtilerman — APPROVED** [review]

**Pull request merged**